kernel/src/scan/log_replay.rs

```diff
@@ -447,15 +443,18 @@ mod tests {
     fn test_no_transforms() {
         let batch = vec![add_batch_simple(get_log_schema().clone())];
         let logical_schema = Arc::new(crate::schema::StructType::new_unchecked(vec![]));
+        let state_info = Arc::new(StateInfo {
+            logical_schema: logical_schema.clone(),
+            physical_schema: logical_schema.clone(),
+            physical_predicate: crate::scan::PhysicalPredicate::None,
+            transform_spec: None,
+        });
```

Comment:
nit: we can just import
```suggestion
let logical_schema = Arc::new(StructType::new_unchecked(vec![]));
        let state_info = Arc::new(StateInfo {
            logical_schema: logical_schema.clone(),
            physical_schema: logical_schema.clone(),
            physical_predicate: PhysicalPredicate::None,
            transform_spec: None,
        });
```